frontend/src/Components/EventCard.jsx

```diff
@@ -4,28 +4,44 @@ import axios from 'axios';
 import moment from 'moment';
 
 
-const EventsCard = (props) => {
+const EventCard = (props) => {
     const history = useHistory();
     const { setFeedback, loggedUser, event } = props;
 
     const [ volunteerHours, setVolunteerHours ] = useState('');
-    const [ volunteersList, setVolunteersList] = useState([]);
+    const [ volunteersList, setVolunteersList ] = useState([]);
+    const [ waitingForRender, setWaitingForRender ] = useState(false);
+
+    /* 
+        props.event.volunteers_list is an array of STRING 
+        where reach element is all one volunteer information related to that event separated by ,
+        if split(', ') we will have:
+            index0: volunteer ID
+            index1: first and last name
+            index2: email
+            index3: volunteer profile deleted
+            index4: event_volunteers table id
+            index5: volunteer confirmed to event
+    */
```

Comment:
nice. glad you did this, my eyes were started to get worried when the index numbers started showing up below

Reply:
documentation ftw